modules/cli-manager-custom-index-krew.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-custom-index_{context}"]
+= Adding the {cli-manager} custom index to Krew
+
+Use the terminal to add the CLI manager custom index to Krew. This is required for the {cli-manager} to function correctly.
+
+[NOTE]
+====
+If you use self-signed certificates, mark the certificate as trusted on your local operating system to use Krew.
+====
+
+.Prerequisites
+
+* link:https://krew.sigs.k8s.io/docs/user-guide/setup/install[Krew is installed].
+* The CLI Manager is installed.
+* You are logged in to {product-title} as a user with the `cluster-admin` role.
+
+.Procedure
+
+. To establish the `ROUTE` variable, enter the following command:
+
+[source,terminal]
+----
+ROUTE=$(oc get route/openshift-cli-manager -n openshift-cli-manager-operator -o=jsonpath='{.spec.host}')
+----
+
+. Name the custom index by running the following command:
+
+[source,terminal]
+----
+# <custom_index_name>=ocp
+----
+
+. To add the custom index to Krew, by entering the following command:
+
+[source,terminal]
+----
+# oc krew index add $<custom_index_name> https://$ROUTE/cli-manager
+----
+
+. To update Krew, enter the following command:
+
+[source,terminal]
+----
+# oc krew update
+----
```

Comment:
May be add a step which reads `make sure no errors are seen when oc kew update` is run.

Reply:
What kind of errors could there be and how would users troubleshoot them?

Reply:
+1 it would be good to show an example too of what a successful output would be after running this command.

Reply:
so a successful run would indicate output something as below, But let me get you a more concrete one may be tomorrow.

I1112 23:55:39.074785 236313 client.go:835] Running 'oc krew update'
  Updated the local copy of plugin index.
  Updated the local copy of plugin index "ocp".

If there is a new plugin detected, sometimes it says that new plugin ocp/oc-mirror detected..